modules/zivid/firmware.py

```diff
@@ -11,7 +11,7 @@ def update(camera, progress_callback=None):
     Call is_up_to_date() first to check if the camera is up to date.
 
     Args:
-        camera: A camera to check if firmware is up to date.
+        camera: The Camera to be updated.
```

Comment:
I do not think upper case is right here (or in the change further down in the file).

If you refer to the class Camera then you must write `Camera instance`, if this is a physical camera then lower case is correct.

If this is referring to the class I do not know if there are any docstring conventions that should be used so that generated code can produce the correct link.

Reply:
You're right, it makes more sense to refer to it as a physical camera in this case, so I changed it to lower-case "camera".

We should look into more docstring conventions when we actually start generating docs, but I don't think that's needed at this point.